test/e2e/storage/vsphere/vsphere_volume_datastore.go

```diff
@@ -32,11 +32,13 @@ import (
 
 const (
 	InvalidDatastore = "invalidDatastore"
+	LocalDatastore   = "localDatastore" // Pre-exists in the testbed
```

Comment:
Instead of defining as const, can we read non-shared local datastore name from environment variable
```
os.Getenv("LOCALDATASTORE") 
```

Reply:
Make sense. Will fix it.

Reply:
Is there are scope for adding this property in our existing conf file? I would prefer us to not to have another environment variable.

Reply:
As discussed with @divyenpatel offline, right now we don't plan to add any test specific environment variables into vsphere.conf file. E2e test will copy and use vsphere.conf file as is without any change.